mangaki/mangaki/settings.py

```diff
@@ -156,6 +156,32 @@
     "allauth.account.auth_backends.AuthenticationBackend"
 )
 
+########################
+# Celery configuration #
+########################
+
+SCHEMA = config.get('celery', 'schema', fallback='redis')
+CELERY_STORE = {
+    'schema': SCHEMA,
+    'host': config.get('celery', '{}_host'.format(SCHEMA), fallback='127.0.0.1'),
+    'port': int(config.get('celery', '{}_port'.format(SCHEMA), fallback=6379)),
+    'password': config.get('secrets', '{}_password'.format(SCHEMA), fallback='')
+}
+
+if SCHEMA == 'redis':
+    CELERY_STORE['database'] = int(config.get('celery', 'redis_database', fallback=0))
+
+if SCHEMA == 'redis':
+    if CELERY_STORE['password']:
+        CELERY_BROKER_URL = "{schema}://:{password}@{host}:{port}/{database}".format(**CELERY_STORE)
+    else:
+        CELERY_BROKER_URL = "{schema}://{host}:{port}/{database}".format(**CELERY_STORE)
+
+    CELERY_RESULT_BACKEND = CELERY_BROKER_URL
```

Comment:
By the way, it means that Redis password will go into `[celery]` rather than `[secrets]`.